src/nupic/research/temporal_memory.py

```diff
@@ -376,12 +376,16 @@ def burstColumn(self, column, columnMatchingSegments, prevActiveCells,
                       `cells`         (iter),
                       `winnerCell`    (int),
     """
+
+    start = self.cellsPerColumn * column
+    cellsForColumn = xrange(start, start + self.cellsPerColumn)
+
     return self._burstColumn(
-      self.connections, self._random,
-      column, columnMatchingSegments, prevActiveCells, prevWinnerCells,
-      self.numActivePotentialSynapsesForSegment,
-      self.cellsPerColumn, self.maxNewSynapseCount, self.initialPermanence,
-      self.permanenceIncrement, self.permanenceDecrement, learn)
+      self.connections, self._random, column, columnMatchingSegments,
+      prevActiveCells, prevWinnerCells,
+      self.numActivePotentialSynapsesForSegment, self.maxNewSynapseCount,
+      self.initialPermanence, self.permanenceIncrement,
+      self.permanenceDecrement, learn, cellsForColumn)
```

Comment:
One quick request: move the parameter up to be after `prevWinnerCells`. This would better follow the scheme:

~~~python
  # These class methods use the following parameter ordering convention:
  #
  # 1. Output / mutated params
  # 2. Traditional parameters to the method, i.e. the ones that would still
  #    exist if this were in instance method.
  # 3. Model state (not mutated)
  # 4. Model parameters (including "learn")
~~~

Oops, typo. "this were in instance method." should be "an", not "in".